Description: $GDM_LANG gets its value from accountsservice's "Language"
 property, and must not be assigned to $LANG in Ubuntu where it typically
 contains a language code (e.g. 'de' or 'en_GB') rather than a valid locale
 name. Besides, this late setting of a locale environment component would make
 it more difficult to fine-tune the settings manually via e.g. ~/.profile.
Forwarded: not-needed
Author: Gunnar Hjalmarsson <ubuntu@gunnar.cc>

Index: gdm-3.5.4.2+git20120803.00576a2d/data/Xsession.in
===================================================================
--- gdm-3.5.4.2+git20120803.00576a2d.orig/data/Xsession.in	2012-08-06 19:08:38.000000000 +0200
+++ gdm-3.5.4.2+git20120803.00576a2d/data/Xsession.in	2012-08-06 19:11:22.000000000 +0200
@@ -153,13 +153,6 @@
 
 unset XKB_IN_USE
 
-# Overwrite $LANG from /etc/profile (and friends) with the one picked in the
-# greeter
-if [ -n "$GDM_LANG" ]; then
-  LANG="$GDM_LANG"
-  export LANG
-fi
-
 xhost +si:localuser:`id -un` || :
 
 # run all system xinitrc shell scripts.
